Jenkinsfile

```diff
@@ -1,21 +1,24 @@
+#!/usr/bin/env groovy
 pipeline {
-    agent any
+  agent none // Allows stages to be built on different labels
 
-    stages {
-        stage('Build') {
-            steps {
-                echo 'Building..'
-            }
+  stages {
+    stage('Tests') {
+      environment {
+        WORKSPACE = "${workspace}"
+        AWS_DEFAULT_REGION = "us-east-1"
+      }
+      agent { label 'linux' }
+      steps {
+        withCredentials([usernamePassword(credentialsId: '8e64020a-fe1f-4fb5-865c-2fcfdacea39e', usernameVariable: 'AWS_ACCESS_KEY_ID', passwordVariable: 'AWS_SECRET_ACCESS_KEY')]) {
```

Comment:
I assume we don't care if random people know what our `credentialsId` is, since this is a public repo

Reply:
yeah it's just an ID, not the creds, that has just seems to be what we had setup in our old Jenkins. We could change it to "super sekret id" and it would look less uuid like.